torch_xla/csrc/tensor.cpp

```diff
@@ -1835,6 +1847,24 @@ Device XLATensor::CommonDeviceForTensors(
   return device;
 }
 
+std::vector<XLATensor> XLATensor::WrapIndicesOnce(
```

Comment:
One of the reasons it's not there is because `XLATensor::Create` is private, which is probably a good thing to preserve. I could take the contents of the `XLATensor` methods returning tensor(s) which are used by `XLATensor::WrapIndicesOnce` and `XLATensor::index`, but having to do that rather suggests this method belongs to `XLATensor`.

Reply:
The index() will be in tensor.cpp .. the creation of the IR Node, in index_op.cpp/h
As well as all its handling.

Reply:
Yes, we can do that at the cost of inlining implementations of `lt` and `where`. They are tiny, it's fine, I'll do it that way.

Reply:
Let me re-check the code ... I am not sure what you mean for inlining ...

Reply:
Done.